apps/cloud_composer/include/pcl/apps/cloud_composer/cloud_view.h

```diff
@@ -70,8 +83,15 @@ namespace pcl
       ProjectModel* 
       getModel () const { return model_; }
       
-      QVTKWidget* 
-      getQVTK() const {return qvtk_; }
+
+#if VTK_MAJOR_VERSION > 8
+      QVTKOpenGLNativeWidget*
```

Comment:
Is it possible to extract out this type change in 1 file?

Reply:
One common file for apps, examples etc?

Reply:
I'm not sure where to put this file, which is common for all projects that require this change, where it makes sense to place it. One place could be in pcl_visualizer, as it is cooperation with the visualizer, but that would add the QVTK* versions as a dependency to pcl_visualizer, which it isn't atm.

Reply:
We could implement and supply a PCL_visualizer-Widget which in turn handles the internal dependencies? Which all subprojects can then use.

Reply:
A simple header in apps would suffice since there's no usage in example (seems that only CMake is updated in example submodule)

Reply:
There are the two tutorials, qt_visualizer and qt_clolorize_cloud, they won't be able to use this header file.